package.json

```diff
@@ -25,9 +25,10 @@
   "homepage": "https://github.com/odysseyscience/react-s3-uploader",
   "dependencies": {
     "aws-sdk": "2.x",
+    "koa-router": "^5.4.0",
```

Comment:
If you are using this router, the assumption would be that you already have a dependency on `koa`.  But if you aren't using this router, you don't want the `koa` dependency, and you would get it if this line exists in `package.json`.  So we should delete this line from here I think.  Make sense?